src/pytorch_lightning/strategies/ipu.py

```diff
@@ -42,28 +41,6 @@
     poptorch = None
 
 
-class LightningIPUModule(_LightningModuleWrapperBase):
-    """
-    .. deprecated:: v1.7.0
-        ``LightningIPUModule`` has been deprecated in v1.7.0 and will be removed in v1.9.0.
```

Comment:
Huh. This says 1.9.0 as a point of removal, while everything else says it should be removed in 1.8.0.

Reply:
Because we changed our opinion to reduce the window and this got missed.